examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -295,15 +300,63 @@ angular.module('cloudberry.common', [])
           }
         }));
 
-        ws.send(sampleJson);
-        ws.send(batchJson);
+        // Used to get timeResult and hashtagResult for complete cache hit case
+        var batchWithoutGeoRequest = (JSON.stringify({
+          batch: [byTimeRequest(parameters), byHashTagRequest(parameters)],
+          option: {
+            sliceMillis: 2000
+          },
+          transform: {
+            wrap: {
+              key: "batchWithoutGeoRequest"
+            }
+          }
+        }));
+
+        geoIdsNotInCache = MapResultCache.returnGeoIdsNotInCache(cloudberryService.parameters.keywords,
```

Comment:
It seems `geoIdsNotInCache` doesn't have to be a global variable?

Reply:
Does not seem to be necessary. Will change it.